### PR TITLE
Update staging.txt

### PR DESCRIPTION
**Pull request by Rkabacoff**
This is a completed book on modern data visualization with R.

### CODE REVIEW

File: R/staging.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+http://rkabacoff.github.io/datavis
```
Comment (jjallaire) [inline R/staging.txt]: Maybe use the https version of the URL here?

**Comment by Rkabacoff** [comment]
Will do. Thank you.

Rob

**Comment by Rkabacoff** [comment]
I made the change. Please let me know if there are any problems.

Thanks
Rob

**Comment by Rkabacoff** [comment]
I noticed that the book cover image was not included, but I have it in YAML. Can you advice what I can do to fix this?

Thank you!

Rob

**Comment by cderv** [comment]
Hi @Rkabacoff , 
You need to use https url only as explained in [About](https://bookdown.org/home/about/). Only https accessible image covers is taken into account. Currently, it seems you use http. Seen in the html source code of your book
```html
<meta property="og:image" content="http://rkabacoff.github.io/datavis/img/cover.png">
```
Just change it to https.

**Comment by Rkabacoff** [comment]
Thank you so much.

Rob

**Pull request merged**